rpcapd/win32-svc.c

```diff
@@ -95,7 +95,7 @@ void WINAPI svc_control_handler(DWORD Opcode)
 				This can be seen by placing a breakpoint at the end of svc_main(), in which you will 
 				see that is never reached. However, as soon as you set the service status to "stopped",
 				the StartServiceCtrlDispatcher() returns and the main thread ends. Then, Win32 has a good
-				authomatic cleanup, so that all the threads which are still running are stopped
```

Comment:
Please submit this, plus "Typos!", as a separate pull request.

Reply:
okay